services/requester/batch_tx_pool.go

```diff
@@ -23,13 +22,7 @@ import (
 
 const (
 	eoaActivityCacheSize = 10_000
-	// Buffer capacity of the channel used for individual transaction
-	// submission. Since `BatchTxPool.Add()` is called quite frequently,
-	// we don't want sending to the channel to block, until there's a
-	// receive ready. That's why we set a high buffer capacity, to allow
-	// `BatchTxPool.Add()` to do its job quickly and release any resources
-	// held, like locks etc.
-	txChanBufferSize = 1_000
+	eoaActivityCacheTTL  = 10 * time.Second
```

Comment:
seems unused?

Reply:
It is used during the creation of the LRU cache:
```go
eoaActivity := expirable.NewLRU[gethCommon.Address, time.Time](
	eoaActivityCacheSize,
	nil,
	eoaActivityCacheTTL,
)
```